src/app/common/modals/comments-modal/comments-modal.tpl.html

```diff
@@ -1,9 +1,14 @@
 <div class="modal-comment">
   <div class="modal-body">
     <div class="image-comment-container">
```

Comment:
Do we need the image content container if it is PDF?

Reply:
I just checked - it should not have been there in the first place. I might have forgotten about that when we submitted the image comments. Removing the div.